### DISCUSSION TITLE
Does it possible to connect two Hub together ?

### DISCUSSION THREAD

**Discussion by Stedewa** [pybricks/support#725] [General]
Hi!

I would like use a second hub to extand my set. I think to use as a slave (Train or Boost Hub) from the master (Technic Hub).

Thanks!

Stéphane

**Comment by tiagozip** [comment]
Hi Stéphane! If could have 2 controllers and connect them via sockets. If you need help with this, check [alto-palo.com/blogs/connect-two-computers-using-secure-socket-programming-in-python](https://alto-palo.com/blogs/connect-two-computers-using-secure-socket-programming-in-python/). For webcam stream, also check [this StackOverflow answer](https://stackoverflow.com/a/61407693/14264568)  
But unfortunately, I didn't find any documentation on how to control the two using pybricks.
Thanks!
Tiago Rangel

**Comment by Nelvarion** [comment]
@dlech @laurensvalk do you plan on extending the lwp3device functionality to work with the pybricks FW?

**Reply by dlech** [reply]
No, it uses a completely different protocol. You can read about what is planned in #262.

**Reply by laurensvalk** [reply]
Do you mean extending this class? https://docs.pybricks.com/en/latest/iodevices/lwp3device.html
